lib/drivers/include/sysctl.h

```diff
@@ -1024,6 +1024,15 @@ uint32_t sysctl_set_spi0_dvp_data(uint8_t en);
  */
 void sysctl_set_power_mode(sysctl_power_bank_t power_bank, sysctl_io_power_mode_t io_power_mode);
 
+/**
+ * @brief       register callback to freq changed
```

Comment:
this comment block doesn't match the code, please fix.

Reply:
Done